lib/systems/src/controllers/NissanModeController.cpp

```diff
@@ -0,0 +1,87 @@
+#include "controllers/NissanModeController.h"
+
+
+DrivetrainCommand_s NissanModeController::evaluate(const VCRData_s &vcr_data, unsigned long curr_millis)
+{
+    DrivetrainCommand_s out = {
+        {0.0f, 0.0f, 0.0f, 0.0f}, // speed rpms
+        {0.0f, 0.0f, 0.0f, 0.0f}  // torques
+    };
+
+    float max_front_power = PhysicalParameters::AMK_MAX_TORQUE  * _front_torque_scale;
+    float max_rear_power = PhysicalParameters::AMK_MAX_TORQUE  * (2 - _front_torque_scale); 
+
+    const PedalsSystemData_s &pedals_data = vcr_data.interface_data.recvd_pedals_data.pedals_data; // get pedals data
+    const InverterData_s &fl_inverter_data = vcr_data.interface_data.inverter_data.FL;
+    const InverterData_s &fr_inverter_data = vcr_data.interface_data.inverter_data.FR;
+    const InverterData_s &rl_inverter_data = vcr_data.interface_data.inverter_data.RL;
+    const InverterData_s &rr_inverter_data = vcr_data.interface_data.inverter_data.RR;
+
+        // Both pedals are not pressed and no implausibility has been detected
+        // accel_request goes between 1.0 and -1.0
+        float accel_request = pedals_data.accel_percent - pedals_data.brake_percent;
```

Comment:
are you doing accel - brake for a specific reason or just because that's how it's done in the simple torque controller? 
we were talking about it and there are drawbacks to this you might want to consider.